app/routes/data.routes.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const DataController = require('../controllers/data/data.controller.js')
+
+const routes = [
+  {
+    method: 'POST',
+    path: '/data/tear-down',
+    handler: DataController.tearDown,
+    options: {
+      description: 'Used to remove the integration test data from the database',
```

Comment:
Potato / potato I know. But we name our UI based test repos 'acceptance tests'. So, that's the name I would use here as well.

 ```suggestion
      description: 'Used to remove the acceptance test data from the database',
```